pkg/controller/workload/kubernetes/target/target.go

```diff
@@ -67,7 +67,7 @@ func Setup(mgr ctrl.Manager, l logging.Logger) error {
 		Complete(&Reconciler{
 			client: resource.ClientApplicator{
 				Client:     mgr.GetClient(),
-				Applicator: resource.NewAPIUpdatingApplicator(mgr.GetClient()),
+				Applicator: resource.NewAPIPatchingApplicator(mgr.GetClient()),
```

Comment:
See this commit message for implications here: https://github.com/crossplane/crossplane/pull/1492/commits/0cd71d2840ab5b1a243a4d18b33002564ef5caf7